vmm/src/vm.rs

```diff
@@ -2675,6 +2675,11 @@ impl Migratable for Vm {
         ]))
     }
 
+    fn start_migration(&mut self) {
+        self.memory_manager.lock().unwrap().start_migration();
```

Comment:
Looks like `start_migration` (also `complete_migration`) does not do anything with `MemoryManager`. Maybe we should remove it from the `vm.start_migration/complete_migration`?

Reply:
I'd leave it there because if we start implementing `start_migration()`/`complete_migration()`, we want to make sure it would be invoked.